Tasks/AzureDevOpsTestPlanV0/getAutomatedTests.ts

```diff
@@ -0,0 +1,105 @@
+﻿import { Console } from "console";
+import tl = require('azure-pipelines-task-lib/task');
+import apim = require('azure-devops-node-api');
+import { WorkItemDetails, TestCase } from 'azure-devops-node-api/interfaces/TestPlanInterfaces';
+import { PagedList } from 'azure-devops-node-api/interfaces/common/VSSInterfaces';
+import TestPlanInterfaces = require('azure-devops-node-api/interfaces/TestPlanInterfaces');
+import VSSInterfaces = require('azure-devops-node-api/interfaces/common/VSSInterfaces');
+
+
+export interface TestPlanData {
+    testPointIds: number[];
```

Comment:
nit: seems testPointIds is not being used, can be removed

Reply:
same reason as above, will be used in future for test run creation and publish test results logic